forge-game/src/main/java/forge/game/ability/effects/CopySpellAbilityEffect.java

```diff
@@ -197,6 +198,15 @@ public boolean test(GameEntity c) {
                         }
                     }
 
+                    if (sa.hasParam("UnearthEffect")) {
```

Comment:
I wonder if it might not be better to remove this and just add the subability to the card with a condition for it being a permanent?

Reply:
I was looking for an existing sample card but couldn't find any. You could guide me how to implement it without adding extra param if it's possible.

Reply:
It may be better to do it this way – sorry I missed the permanent check before

We'll see what Hanmac and TRT think ;)

Reply:
This would Animate on resolving the copy right?
So what happens if you copy that copy again on the stack?

Reply:
I didn't test if the copy would have the animate effect anyway is there such ruling for the copy that gains an effect?

Reply:
Gaining something isn't a copiable value
so if Haste gets applied while the spell wasn't created by this effect it's wrong

Reply:
done, removed nalfeshnee since the copied permanent sub ability is also copied (just tested with lithoform engine)